src/main/java/org/venice/beachfront/bfapi/model/Detection.java

```diff
@@ -3,31 +3,66 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.vividsolutions.jts.geom.Geometry;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
 
 @Entity
+@IdClass(DetectionPK.class)
 @Table(name = "__beachfront__detection")
 public class Detection {
 	@Id
-	@Column(name = "detection_id")
-	@JsonProperty("detection_id")
-	private String detectionId;
-	@Column(name = "job_id")
+	@JoinColumn(name = "job_id", nullable = false, columnDefinition = "VARCHAR(64)")
+	@OnDelete(action = OnDeleteAction.CASCADE)
+	@ManyToOne(targetEntity = Job.class, optional = false)
```

Comment:
I'm thinking manyToOne is appropriate here because for each detection could have multiple jobs?

Reply:
Is the "columnDefinition = "VARCHAR(64)" needed? this may conflict with liquibase should something change in the future? 

I don't think many to one will apply here. One job creates one unique detection. I can't envision a case where this wouldn't be true - especially with our redundant job processing checks. 

If you feel it's "future proofing" for a possible use-case, then we should discuss what that use-case might be and see if it makes sense.

Reply:
Yeah, i probably don't need columnDefinition.

The only reason i was thinking it could be many to one is that it requires a composite key.  If it was one to one, could we just use JobId as the ID?

Reply:
We could use JobId as the ID, or we could have JobId as a property and let Hibernate create its own internal ID. No preference what we use. I would suggest making your decision based on how straightforward and simple your Dao queries end up being. Whichever Dao query makes more sense, I'd go that route.